cmd/frontend/graphqlbackend/virtual_file.go

```diff
@@ -97,3 +97,21 @@ func (r *virtualFileResolver) Highlight(ctx context.Context, args *HighlightArgs
 		Revision: fmt.Sprintf("Preview file diff %s", r.stat.Name()),
 	})
 }
+
+var formatHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+	Name: "virtual_fileserver_format_req",
```

Comment:
I think our metrics usually are prefixed with `src_`

```suggestion
	Name: "src_virtual_fileserver_format_req",
```